codegen.json

```diff
@@ -1,51 +1,30 @@
 {
```

Comment:
+1 Running the script as-is leads to an unclean diff on `codegen.json`, specifically `plugins` being split across lines, and some reordering on `packages/server/graphql/public/resolverTypes.ts`.  Would be a good idea to resolve this before merging.

Reply:
I'll run prettier on it before writing to file

Reply:
PS the reordering is intended. These objects are gonna get looooong so I figure we should sort them for easy manual edits